<?xml version="1.0"?>
<launch>

  <node name="json_prolog" pkg="json_prolog" type="json_prolog" args="perception_ar_kinect" output="screen"/>

  <include file="$(find perception_ar_kinect)/launch/ar_perception.launch"/>
  <include file="$(find perception_ar_kinect)/launch/content_detection.launch"/>
</launch>

